plutus-ledger/src/Ledger/Tx/Internal.hs

```diff
@@ -172,6 +187,55 @@ scriptTxInputs = (\x -> folding x) . filter $ \case
 --     _                                   -> Nothing
 -- inScripts _ _ = Nothing
 
+newtype TxOut = TxOut {getTxOut :: C.TxOut C.CtxTx C.BabbageEra}
+    deriving stock (Show, Eq, Generic)
+    deriving anyclass (ToJSON, FromJSON)
+
+instance C.ToCBOR TxOut where
+  toCBOR (TxOut txout) = C.toCBOR $ C.toShelleyTxOut C.ShelleyBasedEraBabbage txout
+
+instance C.FromCBOR TxOut where
+  fromCBOR = do
+    txout <- C.fromCBOR
+    pure $ TxOut $ C.fromShelleyTxOut C.ShelleyBasedEraBabbage txout
+
+instance Serialise TxOut where
+  encode = C.toCBOR
+  decode = C.fromCBOR
+
+instance NFData TxOut where
+  rnf (TxOut tx) = seq tx ()
+
+instance OpenApi.ToSchema TxOut where
+    declareNamedSchema _ = do
+      addressSchema <- OpenApi.declareSchemaRef (Proxy :: Proxy (C.AddressInEra C.BabbageEra))
+      valueSchema <- OpenApi.declareSchemaRef (Proxy :: Proxy Value)
+      bsSchema <- OpenApi.declareSchemaRef (Proxy :: Proxy Datum)
+      pure $ OpenApi.NamedSchema (Just "TxOut") $ mempty
+        & OpenApi.type_ ?~ OpenApi.OpenApiObject
+        & OpenApi.properties .~
+          [ ("address", addressSchema)
+          , ("value", valueSchema)
+          , ("datum", bsSchema)
+          , ("referenceScript", bsSchema)
+          ]
+        & OpenApi.required .~ ["address","value"]
+
+
+instance Pretty TxOut where
+  pretty (TxOut (C.TxOut addr v d rs)) =
+    hang 2 $ vsep
+      ["-" <+> pretty (fromCardanoTxOutValue v) <> "addressed to"
```

Comment:
<> -> <+>

(btw, you can verify the pretty instances by looking at the file plutus-use-cases/test/Spec/crowdfundingEmulatorTestOutput.txt)